### DISCUSSION TITLE
Ironwail Low GPU Power Consumption

### DISCUSSION THREAD

**Discussion by daftmugi** [andrei-drexler/ironwail#134] [General]
Another feature Ironwail can add to its list as a high-performance QuakeSpasm fork:

- [x] low GPU power consumption

I tested `ad_tears` on Ubuntu 22.04 with a AMD Radeon RX 6000 series card. I chose to record values at one of the most demanding locations.

```
fork                          AMDGPU PPT (±2 W)
---------------               -----------------
Ironwail                       9.00 W
QuakeSpasm                    17.00 W
vkQuake                       20.00 W
```

Also, it's interesting to see CPU temperature with a AMD Ryzen 9 5000 series CPU.

```
fork                          AMDCPU Tctl
---------------               -----------------
Ironwail                      31.4 C
QuakeSpasm                    33.6 C
vkQuake                       38.2 C
```

See screenshot of `ad_tears` location where values were recorded.

![ad_tears power comsumption](https://user-images.githubusercontent.com/102626962/188520231-1d606a7d-5d64-4fce-81a0-fa63cccf99ea.jpg)

This wasn't a rigorous test and just something I found interesting. I wonder what values others get when they test.

**Comment by drHyperion451** [comment]
Did you limit the fps or have vsync enabled? If you leave the fps uncapped the gpu will jump to 100% power usage.

**Reply by daftmugi** [reply]
Yes, it was capped at 60 fps and vsync was on.

Also, power consumption increased (~16 W) in later versions of Ironwail, so there isn't much of a difference anymore these days.
